src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -2266,16 +2266,8 @@ public static void Main(string[] args)
                 var thread1 = new Thread(() => comp.GlobalNamespace.GetMember<NamedTypeSymbol>("A").GetMembers());
```

Comment:
It seems to be forcing us to evaluate the code path in parallel and verify that we updated the result atomically.  It's not perfect but seems to be a healthy excercise of a parallel code path.